inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/config/InstrumentationScopeResolver.java

```diff
@@ -46,12 +42,16 @@
             for (Map.Entry<String, InstrumentationScopeSettings> scopeEntry : source.getScopes().entrySet()) {
                 InstrumentationScopeSettings scopeSettings = scopeEntry.getValue();
 
-                log.debug("Processing scope '{}'.", scopeEntry.getKey());
+                if (log.isDebugEnabled()) {

```

Comment:
I guess below the if check is there for performance reasons, but why is it required here?
"scopeEntry.getKey()" shouldn't really be that performance critical

Reply:
I just didn't want to mix it up whinin one method :P

Reply:
Removed it :)